thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -3737,36 +3737,224 @@ class HashJoinSlaveActivity : public CSlaveActivity, implements IStopInput
 
 //===========================================================================
 
-RowAggregator *mergeLocalAggs(Owned<IHashDistributor> &distributor, CActivityBase &activity, IHThorRowAggregator &helper, IHThorHashAggregateExtra &helperExtra, RowAggregator *localAggTable, mptag_t mptag, bool ordered)
+class CAggregateHT : public CSimpleInterfaceOf<IAggregateTable>
+{
+    struct HTEntry
```

Comment:
Yes agree, I said that in my leading review request comment here: https://github.com/hpcc-systems/HPCC-Platform/pull/9814#issuecomment-292205958

But would prefer to get this into 6.4 and leave that for another JIRA/PR.